src/features/common/types/projectPropContext.d.ts

```diff
@@ -0,0 +1,644 @@
+export interface Coordinates {
```

Comment:
@sunilsabatp The types used here are not related to project props. I see types defined for Viewport, Map etc.

1. Please evaluate whether the type is actually needed. If it is for an external library, it may already be provided by that library.
2. Please separate the type definitions by their purpose into different files.